modules/services/nix-daemon.nix

```diff
@@ -63,7 +63,10 @@ in
 
       serviceConfig.EnvironmentVariables = mkMerge [
         config.nix.envVars
-        { NIX_SSL_CERT_FILE = mkDefault config.environment.variables.NIX_SSL_CERT_FILE;
+        {
+          NIX_SSL_CERT_FILE = mkIf
+            (builtins.hasAttr "NIX_SSL_CERT_FILE" config.environment.variables)
```

Comment:
```suggestion
            config.environment.variables ? NIX_SSL_CERT_FILE
```

Reply:
I think you might still need the brackets around the condition but possibly not the expression after it

Reply:
I want to pass the whole of `mkDefault config.environment.variables.NIX_SSL_CERT_FILE` as the second argument of `mkIf` so I think it is needed.